src/cdk/stepper/stepper.ts

```diff
@@ -71,6 +77,20 @@ export class CdkStep {
   @Input()
   label: string;
 
+  /** Position state of step. */
+  _position: CdkStepContentPositionState;
+
+  /** Sets the CdkStepContentPositionState of step. */
+  set position(position: number) {
```

Comment:
Rather than having a setter for this and reading the property, why not just always read a function like `getAnimationDirection` that's based off of the current index?